src/main/groovy/grails/plugins/airbrake/AirbrakeStackTraceFilterer.groovy

```diff
@@ -0,0 +1,84 @@
+package grails.plugins.airbrake;
+
+import org.grails.exceptions.reporting.DefaultStackTraceFilterer;
+
+
+/**
+ * Created by marc on 11/04/2016.
+ */
+class AirbrakeStackTraceFilterer extends DefaultStackTraceFilterer{
+
+
+    private static final String[] DEFAULT_INTERNAL_PACKAGES = [
```

Comment:
It comes from the original org.grails.exceptions.reporting.DefaultStackTraceFilterer.
This list + packagesToFilter arraylist can probably be removed here.